tutorials/next-chat-application/components/ActiveUsers.js

```diff
@@ -0,0 +1,54 @@
+import { MDBCol, MDBCard, MDBCardBody, MDBTypography } from 'mdb-react-ui-kit';
+
+const ActiveUsers = props => {
+	const { activeUsers } = props;
+	return (
+		<MDBCol md="6" lg="5" xl="4" className="mb-4 mb-md-0">
+			<h5 className="font-weight-bold mb-3 text-center text-lg-start">
+				Active Users
+			</h5>
+			{activeUsers.activeUsers
+				? activeUsers.activeUsers.map(item => (
+						<MDBCard key={Math.random()}>
+							<MDBCardBody>
+								<MDBTypography listUnStyled className="mb-0">
+									<li className="p-2">
+										<a
+											href="#!"
```

Comment:
Should there be a page for the active user? If not you can remove this `a` tag

Reply:
I have modified it.